src/style.css

```diff
@@ -2,0 +1,53 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin: 0;
+}
```

Comment:
Styling the `body` tag directly is not recommended. It's better to create a class for the body or wrap the content in a `div` and style that instead. This way, you can avoid potential conflicts with other CSS libraries or stylesheets that might also style the `body` tag.